Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserTableModel.h

```diff
@@ -57,6 +58,7 @@ namespace AzToolsFramework
         private:
             AZ::u64 m_numberOfItemsDisplayed = 50;
             int m_displayedItemsCounter = 0;
+            int m_updateModelMapTimerId = 0;
```

Comment:
Thanks for the feedback! I really appreciate any info like this. I realize there is a little more bookkeeping needed with using a `timerEvent` like this, but since the timer is getting updated something on the order of 4 or 5 orders of magnitude the idea was to squeeze as much performance out of it as possible. Please correct me if I'm wrong, @[alexmontAmazon](https://github.com/alexmontAmazon).

Reply:
`QTimer` has slightly more overhead, I'd surprised if it showed any meaningful overhead here, but get the emphasis on perf. There's also [`QBasicTimer`](https://doc.qt.io/qt-5/qbasictimer.html) which Qt uses internally that's basically a wrapper that handles exactly what you're doing here.

Reply:
yeah, QBasicTimer pretty much does exactly what Tom's doing here. I prefer to hold the timer ID because it's basically free, but I agree that there won't be any measurable difference between the two.